README.md

```diff
@@ -6,6 +6,10 @@ Storybook is here:
 * Production: <https://ucsb-cs156-w22.github.io/team04-w22-5pm-courses-docs/>
 * QA:  <https://ucsb-cs156-w22.github.io/team04-w22-5pm-courses-docs-qa/>
 
+6pm-2
+* QA dashboard: <https://dashboard.heroku.com/apps/s22-6pm-2-courses-qa/>
+* QA app: <https://s22-6pm-2-courses-qa.herokuapp.com/>
```

Comment:
The code changes look good, but can you add Bryan and Kevin in as collaborators in the Heroku app?
When you're finished, mark this as resolved and re-request my review.

Reply:
Added As Collaborators